README.md

```diff
@@ -63,3 +63,97 @@ Most of the translations can be found in the web client but we have several othe
 <a href="https://translate.jellyfin.org/engage/jellyfin/?utm_source=widget">
 <img src="https://translate.jellyfin.org/widgets/jellyfin/-/jellyfin-web/multi-auto.svg" alt="Detailed Translation Status"/>
 </a>
+
+## Development
```

Comment:
Renamed this to `Server Development`.

I also added another section directory above this describing the purpose of this repository, because I noticed that wasn't actually described anywhere else in the README. I think that should address your other concerns.